tf2onnx/onnx_opset/tensor.py

```diff
@@ -1061,21 +1111,28 @@ def version_1(cls, ctx, node, **kwargs):
 @tf_op("Pack")
 class Pack:
     @classmethod
-    def version_1(cls, ctx, node, **kwargs):
+    def any_version(cls, opset, ctx, node, **kwargs):
         # hack to make up for the missing onnx pack op
         axis = node.get_attr("axis").i
         if axis < 0:
             axis += len(ctx.get_shape(node.input[0])) + 1
 
+        if opset >= 13:
+            node_axes = GraphBuilder(ctx).convert_to_input([axis], "const_axes", is_optional=True, dtype=np.int64)
+
         inputs = []
         dtype = None
         # insert Unsqueeze on each input
         for i, n in enumerate(node.inputs):
             dtype = ctx.get_dtype(node.input[i])
             shape = ctx.get_shape(node.input[i]).copy()
             shape.insert(axis, 1)
-            new_node = ctx.make_node("Unsqueeze", [node.input[i]], op_name_scope=node.name, attr={"axes": [axis]},
-                                     shapes=[shape], dtypes=[dtype])
+            if opset < 13:
```

Comment:
Can you remove this if/else and use graphbuilder in all cases?

Reply:
In that case, I chose to have only one constant for the axes for all squeeze node. I assume duplicated constants get removed.

Reply:
Yeah duplicated constants are optimized out.